### PR TITLE
FWF-3846: [Feature] Added style for expression-builder modal

### PR DESCRIPTION
**Pull request by Bonymol-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes
*  Added style for expression builder
*  Added tick icon in svg

### CODE REVIEW

File: forms-flow-theme/scss/_modal.scss [hunk 1]
```diff
@@ -593,3 +593,202 @@
     border: none;
   }     
 }
+
+
+//expression-builder-modal
+
+  .expression-list-group {
```
Comment (arun-s-aot) [inline forms-flow-theme/scss/_modal.scss:600]: please do scss nesting of classes.

File: forms-flow-theme/scss/_modal.scss [hunk 2]
```diff
@@ -593,3 +593,202 @@
...
+    font-size: var(--ff--font-size-sm);
+    font-weight: var(--font-weight-lg);
+  }
+
+  .expression-list-group .variable-key {
+    color: var(--ff-gray-400);
+    font-size: var(--ff--font-size-sm);
+    font-weight: var(--font-weight-sm);
+    margin-left: 8px;
+  }
```
Comment (arun-s-aot) [inline forms-flow-theme/scss/_modal.scss:638]: --ff--font-size-sm variable dont exist . please use --font-size-sm in both cases

File: forms-flow-theme/scss/_modal.scss [hunk 3]
```diff
@@ -593,3 +593,202 @@
...
+    display: flex;
+    padding: var(--spacer-200) var(--spacer-200);
+    flex-direction: column;
+    align-items: flex-start;
+    gap: var(--spacer-150);
+  }
+
+  .no-var-text {
+    color: var(--ff-gray-400);
+    font-weight: var(--ff-font-weight-sm);
```
Comment (arun-s-aot) [inline forms-flow-theme/scss/_modal.scss:738]: --ff-font-weight-sm dont exist. pls check

File: forms-flow-theme/scss/_modal.scss (hunk 1 above)
Comment (Bonymol-aot) [reply forms-flow-theme/scss/_modal.scss:600]: Added

File: forms-flow-theme/scss/_modal.scss (hunk 2 above)
Comment (Bonymol-aot) [reply forms-flow-theme/scss/_modal.scss]: Resolved

File: forms-flow-theme/scss/_modal.scss (hunk 3 above)
Comment (Bonymol-aot) [reply forms-flow-theme/scss/_modal.scss]: Resolved

**Review by Ajay-aot — APPROVED** [review]

**Review by arun-s-aot — APPROVED** [review]

**Pull request merged**